lib/puppet/provider/firewalld_rich_rule/firewall_cmd.rb

```diff
@@ -26,6 +28,7 @@ def key_val_opt(opt, resource_param = opt)
   def eval_source
     args = []
     return [] unless (addr = @resource[:source])
```

Comment:
Unrelated, but I wonder why this isn't either `return args ...` or `return [] ...` and then `args = []`. Now it's just creating an unused array in some cases.